docs/dev.rst

```diff
@@ -33,12 +39,30 @@ This will perform several tasks for you. It will:
 
 * run service initialization scripts, such as service user creation and Waffle configuration.
 
+Additionally, if you chose to bind-mount your local clone of edx-platform, it will:
+
+* re-run setup.py,
+
+* clean-reinstall Node modules, and
+
+* regenerate static assets.
+
 Once setup is complete, the platform will be running in the background:
 
 * LMS will be accessible at `http://local.overhang.io:8000 <http://local.overhang.io:8000>`_.
 * CMS will be accessible at `http://studio.local.overhang.io:8001 <http://studio.local.overhang.io:8001>`_.
 * Plugged-in services should be accessible at their documented URLs.
 
+Now, you can use the ``tutor dev ...`` command-line interface to manage your development environment. Some common commands are described below.
```

Comment:
I'd like to limit the use of "you" or "you can" as much as possible in the docs. I understand it's sometimes quite difficult, but I think that a more direct phrasing is more appropriate. It's not really a matter of "you" vs "we" vs "one" pronouns, but more of conciseness.

For instance:

> Now, run the ``tutor dev ...``` command line interface to manage the development environment.

> Wherever the ``[--mount=./edx-platform]`` option is present, either:
>
>   * omit it when running off the edx-platform repository built into the image, or
> * substitute it...

> Once ``launch`` was run once...

> Read more about bind-mounts :ref:`below <bind_mounts>`.

> To debug a local edx-platform repository, add a python breakpoint...

I'm not very strict about this rule. It's really just a general recommendation. If you disagree it's totally fine.

Reply:
Good point. I tried that out, and the document does feel more concise now.